docs/sources/features/datasources/loki.md

```diff
@@ -111,11 +124,16 @@ log message you're interested in.
 
 ## Templating
 
-Template variables are not yet supported by Loki.
+Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns makes it easy to change the data being displayed in your dashboard.
```

Comment:
Missing `r`
> ... in your metric ...

Missing comma
> metric queries, you can use 

Drop-down box is hyphenated
> ... shown as drop-down boxes ...

Avoid using drop-down as a noun:
> These drop-down boxes makes it ...